sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -588,7 +654,7 @@ Write withBigtableService(BigtableService bigtableService) {
     public PDone expand(PCollection<KV<ByteString, Iterable<Mutation>>> input) {
       getBigtableConfig().validate();
 
-      input.apply(ParDo.of(new BigtableWriterFn(getBigtableConfig().getTableId(),
+      input.apply(ParDo.of(new BigtableWriterFn(getBigtableConfig().getTableId().get(),
```

Comment:
This will be resolved at pipeline construction time effectively making it so that static value providers can only be used.

Reply:
Addressed